### PR TITLE
Update HLS collection sql script

### PR DESCRIPTION
**Pull request by jvntf**
Add properties to SQL script as per [conventions](https://github.com/NASA-IMPACT/delta-backend/issues/29)

Only adds `is_periodic` and `time_density`, and summaries. in summaries ive added the date range for what has been ingested so far... is this correct? Or should this be a list of all the records that had been ingested? In that case, seems like we would want to insert this collection with summaries empty and then populate it later.

### CODE REVIEW

File: scripts/collections-sql/HLSS30.002.sql [hunk 1]
```diff
@@ -4,6 +4,9 @@ INSERT INTO pgstac.collections (content) VALUES('{
    "links":[
    ],
    "title":"HLSS30.002",
+    
+   "dashboard:is_periodic": True,
+   "dashboard:time_density":
```
Comment (abarciauskas-bgse) [inline scripts/collections-sql/HLSS30.002.sql:9]: We don't want this, just in properties

File: scripts/collections-sql/HLSS30.002.sql [hunk 2]
```diff
@@ -38,7 +41,14 @@ INSERT INTO pgstac.collections (content) VALUES('{
...
    }],
-   "stac_version":"1.0.0"
+   "stac_version":"1.0.0",
+   "properties": {
+     "dashboard:is_periodic": true
+     "dashboard:time_density": "hour",
+     "summaries": {
+       "datetime": ["2021-07-29 01:00:00","2021-07-29 05:00:00Z"]
+     }
+   }
```
Comment (abarciauskas-bgse) [inline scripts/collections-sql/HLSS30.002.sql:51]: I think we also need `item_assets` right @anayeaye ?

**Comment by anayeaye** [comment]
I'm late here but I think we can leave HLS as it was created in dev, it's another animal and we probably need to come up with a different way to handle it in the front end...

**Comment by abarciauskas-bgse** [comment]
Ah ok then @jvntf sorry for the effort but probably you can try updating the stack to point to the new database for staging and inserting the HLS collection and item records there. Again I think we just want spatial and temporal coverage for the Hurricanes.

**Comment by abarciauskas-bgse** [comment]
Closing this as stale and I think we want to revisit the HLS approach

**Pull request closed without merging**